lib/routes/routePUT.js

```diff
@@ -38,6 +39,8 @@ export default function routePUT(request, response, log, statsClient) {
         request.post = '';
         const post = [];
         let postLength = 0;
+        // Send continue message for streaming V4 requests, if applicable.
+        writeContinue(request, response);
```

Comment:
If this is only for streaming v4, can you put it in the route if condition for just when there is an object key (so put object) or put object part?